adafruit_tca9548a.py

```diff
@@ -32,6 +32,14 @@
 import time
 from micropython import const
 
+try:
+    from typing import Optional, List  # pylint: disable=unused-import
```

Comment:
I think we can get rid of `Optional` and the pylint disable comment here. 

The "unused" import is only needed in files where there are not any typing specific imports in use.

In this file we have `List` in use which fulfills our purpose of raising the ImportError on microcontrollers.

Reply:
oops - bad housekeeping on my part!